### ISSUE TITLE
SRW - Implement Cylindrical Fiber element in Sirepo

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#181]
Need to implement Cylindrical Fiber element based on `srwl_opt_setup_cyl_fiber()` function from `srwlib.py`.

**Comment by mrakitin** [comment]
It's necessary to finish #182.

**Comment by mrakitin** [comment]
```python
    #Boron Fiber (with Tungsten core)
    if('FIB' in arElNames):
        fpln = 3 #focusing in both planes
        if((_v.op_FIB_fpl == 'h') or (_v.op_FIB_fpl == 'H') or (_v.op_FIB_fpl == 'x') or (_v.op_FIB_fpl == 'X')): fpln = 1
        elif((_v.op_FIB_fpl == 'v') or (_v.op_FIB_fpl == 'V') or (_v.op_FIB_fpl == 'y') or (_v.op_FIB_fpl == 'Y')): fpln = 2

        el.append(srwl_opt_setup_cyl_fiber(fpln, _v.op_FIB_delta_e, _v.op_FIB_delta_c, _v.op_FIB_atnl_e, _v.op_FIB_atnl_c, _v.op_FIB_d_e, _v.op_FIB_d_c, _v.op_FIB_x, _v.op_FIB_y))
        pp.append(_v.op_FIB_pp)
```

**Comment by mrakitin** [comment]
```python
    ['op_FIB_fpl', 's', '', 'FIB: focusing plane ("h" or "v" or "hv" or "")'],
    ['op_FIB_delta_e', 'f', 4.20756805e-06, 'Fiber: refractive index decrement of main (exterior) material'],
    ['op_FIB_delta_c', 'f', 4.20756805e-06, 'Fiber: refractive index decrement of core material'],
    ['op_FIB_atnl_e', 'f', 7312.94e-06, 'Fiber: attenuation length of main (exterior) material [m]'],
    ['op_FIB_atnl_c', 'f', 7312.94e-06, 'Fiber: attenuation length of core material [m]'],
    ['op_FIB_d_e', 'f', 100.e-06, 'Fiber: ext. diameter [m]'],
    ['op_FIB_d_c', 'f', 10.e-06, 'Fiber: core diameter [m]'],
    ['op_FIB_x', 'f', 0., 'Fiber: horizontal center position [m]'],
    ['op_FIB_y', 'f', 0., 'Fiber: vertical center position [m]'],
```

**Comment by mrakitin** [comment]
Sirepo now generates the code like:
```python
def set_optics(v=None):
    el = []
    el.append(srwlib.srwl_opt_setup_cyl_fiber(_foc_plane='hv', _delta_ext=5.555666e-06, _delta_core=3.645069e-05, _atten_len_ext=0.00286169, _atten_len_core=4.21889e-06, _diam_ext=0.0001, _diam_core=1e-05, _xc=0.0, _yc=0.0))
    el.append(srwlib.SRWLOptD(1.0))
```

**Comment by mrakitin** [comment]
Implemented in 278e142. Closing.

**Comment by mrakitin** [comment]
Forgot to add title and position fields.

**Comment by mrakitin** [comment]
Corrected in f11a293. Closing.
